src/main.ts

```diff
@@ -1,8 +1,28 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { HttpExceptionFilter } from './common/filter/http-exception.filter';
+import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const allowedOrigins = process.env.ALLOWED_ORIGINS.split(',');
+
```

Comment:
```suggestion
  app.useGlobalInterceptors(new ClassSerializerInterceptor(app.get(Reflector)));
```
위 코드를 혹시 추가해주실 수 있을까요? 응답객체 직렬화에 사용합니다.
(참고: https://jojoldu.tistory.com/610)

Reply:
네 참고링크가 많은 도움되었습니다! 이 부분도 추가해보겠습니다

Reply:
main.ts에서 글로벌 인터셉터를 추가했습니다